CHANGELOG.md

```diff
@@ -6,7 +6,8 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 ## Latest
-* Update Roslyn to 4.9.0-3.23611.3 (PR: [#<>](https://github.com/dotnet/vscode-csharp/pull/<>))
+* Update Razor to 7.0.0-preview.23611.2 (PR: [#6741](https://github.com/dotnet/vscode-csharp/pull/6741)
```

Comment:
this will need to go into a different section since we snapped yesterday.  Can use a placeholder version and I'll fix it up once we have the real one.